lib/proxy_api/template.rb

```diff
@@ -10,9 +10,7 @@ def template_url
       if (response = parse(get("templateServer"))) and response["templateServer"].present?
         return response["templateServer"]
       end
-    rescue SocketError, Timeout::Error, Errno::EINVAL, Errno::ECONNRESET,
-           EOFError, Net::HTTPBadResponse, Net::HTTPHeaderSyntaxError,
-           Net::ProtocolError, RestClient::ResourceNotFound => e
+    rescue Exception => e
```

Comment:
We shouldn't `rescue Exception`, that would catch even things like SyntaxError, SystemExit. We should rescue StandardError or avoid using the exception class which defaults to StandardError. I prefer listing only exceptions that I expect so I think this patch does not fix the problem correctly. I'd recommend adding the exception that that was not rescued to the list instead.